README.md

```diff
@@ -1,39 +1,33 @@
 # MIxS: Minimum Information about any (X) Sequence
 
-⚠️ ⚠️ ⚠️ 
+MIxS is a specification for the Minimum Information about any (X) Sequence standard.
```

Comment:
This sounds more like X and Y are synonyms. So what is MIxS really a specification for? That is an extremely tricky question.

Reply:
I would suggest something more like

MIxS, or the Minimum Information about any (X) Sequence is a specification for describing the provenance of a nucleotide sequence:
- what do we know about the sample it came from?
- in what environment was the sample found?
- in what ways were the nucleic acids extracted, amplified, etc from the sample?